src/app/shared/services/tv2-output-layer.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Part } from '../../core/models/part'
+import { Tv2OutputLayer } from '../../core/models/tv2-output-layer'
+import { Tv2Piece } from '../../core/models/tv2-piece'
+
+export class Tv2OutputLayerService {
```

Comment:
Do we technically need the `Tv2...` part of the name? 
All of the frontend i technically ours.
Same goes for other types which has had `Tv2` prefixed to it.

Reply:
For now we want to keep it easy to fork and start working on for other Sofie collaborators. There are some things that need clearer separation for this to ideal, but having the Tv2 prefix for our specific code helps with that. And it keeps it the types matching between the frontend and blueprints.